test/fixtures/es-module-loaders/builtin-named-exports-loader.mjs

```diff
@@ -23,22 +24,22 @@ export function resolve(specifier, context, defaultResolve) {
   return def;
 }
 
-export function getSource(url, context, defaultGetSource) {
+export function getSource(url) {
   if (url.startsWith('custom-nodejs:')) {
     const urlObj = new URL(url);
     return {
       source: generateBuiltinModule(urlObj.pathname),
       format: 'module',
     };
   }
-  return defaultGetSource(url, context);
+  return null;
```

Comment:
🎉

Reply:
I'm pretty sure we need these default hooks available.

Reply:
We're currently discussing opening this up in a subsequent PR. For now the `default` -> `next` transition is what is implemented here, which was actually the original design laid down when we first discussed the `default` style hooking pattern.